lib/features/login/widgets/welcome_view.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:gruene_app/app/theme/theme.dart';
+import 'package:gruene_app/features/login/bloc/auth_bloc.dart';
+import 'package:gruene_app/i18n/translations.g.dart';
+
+class WelcomeView extends StatelessWidget {
```

Comment:
Not sure about whether to extract this to an own widget and if yes, on which naming to use. Do you have any suggestions?